bika/lims/api.py

```diff
@@ -1052,6 +1052,37 @@ def get_current_user():
     return ploneapi.user.get_current()
 
 
+def get_user_contact(user, contact_types=None):
+    """Returns the current contact associated to the user passed in, if any.
+    If the user passed in has no contact associated, return None. If
+    contact_types is not specified, searches for 'Contact' and 'LabContact'
+    :param: Plone user
+    :contact_types: array with the contact types to look for
+    :returns: Contact associated to the Plone user, if any. Otherwise, None
+    """
+    if not user:
+        return None
+
+    ctypes = ['Contact', 'LabContact']
+    if contact_types:
+        ctypes = contact_types
+    user_id = user.id
+    pc = get_tool('portal_catalog')
+    brains = pc(portal_type = ctypes, getUsername = user_id)
```

Comment:
- Better use the `search` function here to handle all catalogs
- Spaces between the `=` operator are never used for keyword arguments